checks/namespace.go

```diff
@@ -156,7 +155,7 @@ func (c *NamespaceChecker) Check(extConfig external.Check) *pkg.CheckResult {
 	if _, err := pods.Create(context.TODO(), pod, metav1.CreateOptions{}); err != nil {
 		return unexpectedErrorf(check, err, "unable to create pod")
 	}
-	pod, err = c.WaitForPod(ns.Name, pod.Name, time.Millisecond*time.Duration(check.ScheduleTimeout), v1.PodRunning)
+	pod, _ = c.WaitForPod(ns.Name, pod.Name, time.Millisecond*time.Duration(check.ScheduleTimeout), v1.PodRunning)
```

Comment:
Should this error not be handled?

Reply:
This is a flakey one... I tried to handle this error but the namespace test started to fail, even in case of error we want to proceed. and check further steps

Reply:
What about just logging it?